docs/source/top_level_elements/name.rst

```diff
@@ -0,0 +1,350 @@
+
+Names with URIs
+---------------
+
+We have at least 19,670 records with names with matching valueURIs.
+
+If a name has a URI, we should use it as the object like the name in
+`harp:1 <https://digital.lib.utk.edu/collections/islandora/object/harp%3A1/datastream/MODS>`_:
+
+.. code-block:: xml
+
+    <name authority="naf" valueURI="http://id.loc.gov/authorities/names/no2002022963">
+        <namePart>Swan, W. H. (William H.)</namePart>
+        <role>
+            <roleTerm authority="marcrelator" valueURI="http://id.loc.gov/vocabulary/relators/cmp">
+                Composer
+            </roleTerm>
+        </role>
+        <role>
+            <roleTerm authority="marcrelator" valueURI="http://id.loc.gov/vocabulary/relators/com">
+                Compiler
+            </roleTerm>
+        </role>
+    </name>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+
+    <https://example.org/objects/1>
+        relators:cmp <http://id.loc.gov/authorities/names/no2002022963> ;
+        relators:com <http://id.loc.gov/authorities/names/no2002022963> .
+
+
+Names without URIs
+------------------
+
+We have at least 31,618 records without names with matching valueURIs.
+
+If the name does not have a URI, we can just use the string literal of namePart as the value.
+
+An example can be found in `cDanielCartoon:1178 <https://digital.lib.utk.edu/collections/islandora/object/cDanielCartoon%3A1178/datastream/MODS>`_:
+
+.. code-block:: xml
+
+    <name type="personal">
+        <namePart>Daniel, Charles R. (Charlie), Jr., 1930-</namePart>
+        <role>
+            <roleTerm type="text" authority="marcrelator" valueURI=" http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+    </name>
+
+We would just model this as:
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+
+    <https://example.org/objects/1>
+        relators:cre "Daniel, Charles R. (Charlie), Jr., 1930-" .
+
+Names with @usage="primary"
+---------------------------
+
+There are 314 records with an @usage="primary" attribute.  An example is
+`kefauver:150412002 <https://digital.lib.utk.edu/collections/islandora/object/kefauver%3A150412002/datastream/MODS>`_.
+
+.. code-block:: xml
+
+    <name usage="primary">
+        <namePart>unknown</namePart>
+        <role>
+            <roleTerm authority="marcrelator" type="text" valueURI="http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+    </name>
+
+We should drop this.
+
+Namepart with @type="date"
+--------------------------
+
+There are 10,370 records with a namePart with a @type="date".  This value seems to relate to the birth and death dates of
+the person.
+
+An example is `egypt:230 <https://digital.lib.utk.edu/collections/islandora/object/egypt%3A230/datastream/MODS>`_
+
+If we need to keep this data, we need to have an external triple store and decide how to model names.
```

Comment:
One thing I've been trying to think through is whether the string we display and the actual preferred label associated with an entity need to match. I've been thinking about this a lot for Wikidata too. For instance, we transform names with dates for display in Islandora so that the name/label and the date is a single string - "Sébah, Jean Pascal" becomes "Sébah, Jean Pascal, 1872-1947". We could've just had a displayForm and avoided some of this. Ultimately all of these names with dates should have a URI though, so the string itself is kind of moot. While I think the dates may provide helpful context to the user, the user can take the additional effort to look individuals up.

Reply:
@markpbaggett I vaguely recall talking about working towards leveraging/parsing values from a given valueURI in this type of case. Do we need to talk about that here at all (just a brief mention) or should that articulation live in the mapping document?

Reply:
@CanOfBees does this address your request?

Reply:
@markpbaggett Yes - thanks for adding.